### PR TITLE
FXIOS-1069 ⁃ Issue7500-XCUIPerformanceTabsTestFollowUp

### LINKED ISSUE #7500: FXIOS-1059 ⁃ Follow up: Performance Tabs Test
This issue is to track some pending work after PR #7486 landed:
- testPerfTabs20startupt: just realized it does not have DB associated to show the expected tabs
- create doc to explain these tests
- a bit of clean up for the nit comments

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-1059)

### PR DESCRIPTION
**Pull request by isabelrios**
Fixes #7500

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-1069)

### CODE REVIEW

**Comment by isabelrios** [comment]
@AaronMT @rpappalax please take a look at the doc I created to expalin about the performance tests added...any feedback is more than welcome... Thanks!

File: Docs/performance-tests.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+Performance tests
+==================
+
+There are several scenarios in which having performance tests to meassure how the app performs is really important. The data bases for Bookmarks, History, Tabs could be very big and impact in the time it takes to launch the app or use it. 
+
+Being able to pre-load existing data bases with heavy profiles has helped to implement these tests. It is possible to launch a test with a specific profile and meassure differnt data that could help to find issues for the app in that situation.
+
```
Comment (AaronMT) [inline Docs/performance-tests.md:7]: s/data bases/databases

File: Docs/performance-tests.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* Disk locigal writes (kB)
+
+
+As a first approach the data is being recorded in a [doc](https://docs.google.com/spreadsheets/d/1ERhNf1IY7Rqfzvcvb2PJldUiHWNArh81yEx6rJ-BA-U/edit?usp=sharing) after running the tests on pyhiscal device.
+
+The tests are using a low size profile (with 20 tabs) and a heavy profile (with 1280 tabs), there are two scenarios covered for each profile:
+* Launch app
+* Open Tabs Tray
+
+It was decided to use those edge profiles and maybe add more in the future depending on the data retrieved and how the app performs.
```
Comment (AaronMT) [inline Docs/performance-tests.md:40]: edge profile?

**Review by AaronMT — APPROVED** [review]

**Comment by AaronMT** [comment]
Thanks for the markdown write-up.

File: Docs/performance-tests.md (hunk 2 above)
Comment (isabelrios) [reply Docs/performance-tests.md:40]: wrong translation :( I wanted to say like extreme profiles but trying to use a different word

File: Docs/performance-tests.md (hunk 1 above)
Comment (isabelrios) [reply Docs/performance-tests.md:7]: sorry not sure where should I change/add that part... in Line 6 it says data bases as the second option in your comment...

**Comment by isabelrios** [comment]
Landing since @rpappalax will be on PTO for two weeks, we can re-visit the documentation later on as well as the test suite.

**Pull request merged**